{{cookiecutter.github_repository}}/provisioner/roles/nginx/templates/site.80.conf.j2

```diff
@@ -12,7 +12,7 @@ server {
 
     {% if vm and (nginx_cert.stat.exists == false or nginx_key.stat.exists == false) %}
     location / {
-        uwsgi_pass unix:///tmp/django-{{ domain_name }}-uwsgi.sock;
+        uwsgi_pass unix:///tmp/django-uwsgi.sock;
```

Comment:
This was changed unnecessarily and broke the VM execution. The project wise process, logs, service names is coming in separate PR by @vikalpj